lib/board.rb

```diff
@@ -0,0 +1,19 @@
+# Understands how to represent a board for tic-tac-toe
```

Comment:
Domain design: This definition feels circular because it mentions the class.  What does a board do?